src/api.js

```diff
@@ -0,0 +1,30 @@
+
+let api = {
```

Comment:
this part is GREAT. i had a feeling we'd have to put the APIs in their own file, but having them wrapped into an object for easy accessibility is great.